pimpmycause/profiles/models.py

```diff
@@ -1,26 +1,69 @@
 from __future__ import unicode_literals
 
 from django.db import models
-from django.contrib.auth.models import User
```

Comment:
While this does store the email for the user by default it doesn't use the email as the username. So the third party app was a simpler way and recommended to me over overriding the django default by hand. If we were to simply "extend" the user in the admin we get 2 User panels one for default user and the default fields and a second one for the custom Pimp user and the extra fields we added. 
Now we have only 1 user in the admin which is just the Pimp user and no confusing multiple user things going on.